tests/cypress/integration/actions_objects/case_99_save_filtered_object_in_AAM.js

```diff
@@ -0,0 +1,49 @@
+// Copyright (C) 2021 Intel Corporation
+//
+// SPDX-License-Identifier: MIT
+
+/// <reference types="cypress" />
+
+import { labelName, taskName } from '../../support/const';
+
+context('Save filtered object in AAM.', () => {
+    const caseId = '99';
+    const newLabelName = `New label for case ${caseId}`;
+    const createCuboidShape2Points = {
+        points: 'From rectangle',
+        type: 'Shape',
+        labelName: labelName,
+        firstX: 250,
+        firstY: 350,
+        secondX: 350,
+        secondY: 450,
+    };
+
+    before(() => {
+        cy.openTask(taskName);
+        cy.addNewLabel(newLabelName);
+        cy.openJob();
+        cy.createCuboid(createCuboidShape2Points);
+    });
+
+    describe(`Testing case "${caseId}"`, () => {
+        it(`Set filter label == “${labelName}”.`, () => {
+            cy.addFiltersRule(0);
+            cy.setFilter({
+                groupIndex: 0,
+                ruleIndex: 0,
+                field: 'Label',
+                operator: '==',
+                value: labelName,
+                submit: true,
+            });
+        });
+
+        it(`Go to AAM and change a label for the shape. Save the changes. UI is not failed.`, () => {
+            cy.changeWorkspace('Attribute annotation', newLabelName);
```

Comment:
I am not sure changing label should be a part of ``cy.changeWorkspace``. It looks a bit strange and I understood why it should work only after I open ``cy.changeWorkspace`` implementation

Reply:
I separated these commands and adapted the corresponding tests.